### ISSUE TITLE
"Removes all built-in apps" but removes Minecraft Bedrock

### ISSUE THREAD

**Issue by EndxAwesome14** [mirinsoft/spydish#62]
Just wanted to warn people, it will delete things that aren't built-in to your Windows also. I regret choosing that option, thinking that I wouldn't lose 200+ hours of my life.

**Comment by technoabyss** [comment]
It took your saves?

**Comment by jddcef** [comment]
Ouch, it does remove all "Minecraft*" apps, and some others too... Perhaps it should be more specific...
Generally the script is probably meant more for unused or cleaner/newer machines/builds... 
 https://github.com/builtbybel/privatezilla/blob/0c24b54027311ddec8bf81aa6069b7b2da8809f9/scripts/Run%20Windows10Debloater%20(Community).ps1#L98-L116

**Comment by jddcef** [comment]
Is there no filter one can use to make sure one is removing only 
1) Unused/never opened apps 
OR 
2) Silently installed apps, i.e. apps that weren't user initiated installations
OR 
3) Apps that have not been used recently or frequently
OR
4) Has no user specific saved data

This would reduce chances of removing an app that is actually in use or with some user data

**Comment by chitinlink** [comment]
I've just suffered some serious data loss thanks to this feature, it uninstalled my WSL distro and reinstalling it overwrote my files. Please change this feature to include a list of affected apps or some kind of warning.

**Comment by avelican** [comment]
Yeah, looks like Minecraft keeps its world data in the UWP app folder:

C:\Users\you\AppData\Local\Packages\Microsoft.MinecraftUWP_8wekyb3d8bbwe\LocalState\games\com.mojang\

So when the app is uninstalled (as happens when you select PrivateZilla's "Remove all built-in apps" feature) you lose the world data too. Sad!

The UI says "all built in apps", and Minecraft is not built-in...

It would also be nice if it showed you a list of everything it's going to remove, and maybe double check for confirmation? I checked the source code for Privatezilla and didn't see anything I'd miss in the apps list. I didn't realize it would delete literally everything! That should probably be specified more clearly in the UI.
